### PR TITLE
Modified proposer and committee string value to address

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- Updated proposer and committee string value to `common.Address` type.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ehnuje — DISMISSED** [review]

**Review by KimKyungup — APPROVED** [review]

File: datasync/chaindatafetcher/kafka/utils.go [hunk 1]
```diff
@@ -28,35 +26,31 @@ import (
...
 	for i, addr := range istanbulExtra.Validators {
 		commiteeAddrs[i] = addr
 	}
-	var strValidators []string
-	for _, validator := range istanbulExtra.Validators {
-		strValidators = append(strValidators, validator.Hex())
-	}
 
-	return proposerAddr.Hex(), strings.Join(strValidators, ","), nil
+	return proposerAddr, commiteeAddrs, nil
```
Comment (aidan-kwon) [inline datasync/chaindatafetcher/kafka/utils.go:53]: commiteeAddrs -> committeeAddrs

Comment (aidan-kwon) [inline datasync/chaindatafetcher/kafka/utils.go:53]: Why don't you return `istanbulExtra.Validators`?

Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kafka/utils.go]: Removed this variable name.

Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kafka/utils.go]: My bad. Updated. 1d2868f

**Review by aidan-kwon — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Review by ehnuje — APPROVED** [review]

**Review by KimKyungup — APPROVED** [review]

**Pull request merged**